src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -527,6 +527,41 @@ public String toString() {
       return "List<StreamEntry>";
     }
   };
+
+  public static final Builder<StreamEntry> STREAM_ENTRY = new Builder<StreamEntry>() {
+    @Override
+    @SuppressWarnings("unchecked")
+    public  StreamEntry build(Object data) {
+      if (null == data) {
+        return null;
+      }
+      ArrayList<Object> objectList = (ArrayList<Object>) data;
```

Comment:
Shouldn't it be `List` instead of ArrayList? Under the hood, it is ArrayList but it isn't *guaranteed* to be so.

Reply:
done